src/components/Dialogs/Dialogs.js

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 
 import DialogItem from './DialogsItem/DialogsItem';
 import Message from './Message/Message';
+import { updateNewMessageBodyCreator, sendMessageCreator } from '../../redux/state';
 
 import style from './Dialogs.module.css';
 
 const Dialogs = (props) => {
-  const { dialogs } = props.state;
-  const { messages } = props.state;
+  const state = props.store.getState().dialogsPage;
+  const dialogsElements = state.dialogs.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
+  const messagesElements = state.messages.map((message) => <Message message={message.message} />);
+  const { newMessageBody } = state;
 
-  const dialogsElements = dialogs.map((dialog) => <DialogItem name={dialog.name} id={dialog.id} />);
-  const messagesElements = messages.map((message) => <Message message={message.message} />);
-  const newDialogText = React.createRef();
+  const onSendMessageClick = () => {
+    props.store.dispatch(sendMessageCreator());
+  };
 
-  const showDialogText = () => {
-    const text = newDialogText.current.value;
-    alert(text);
+  const onNewMessageChange = (e) => {
+    const body = e.target.value;
+    props.store.dispatch(updateNewMessageBodyCreator(body));
   };
 
   return (
```

Comment:
Consider breaking this return JSX into several functions inside the current function and then calling them in the return statement.

Reply:
/fxd